src/karmen_backend/Dockerfile

```diff
@@ -1,41 +1,25 @@
-FROM python:3.7-alpine as builder
-RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+FROM python:3.7-slim
 
-RUN mkdir /install
-WORKDIR /install
+# ensure www-data user
+# RUN set -x ; \
+#   groupadd --gid 82 --system www-data && \
+#   useradd --gid 82 --defaults --system -G www-data www-data && exit 0 || exit 1
 
-# build deps
-RUN apk --update add --no-cache gcc make postgresql-client postgresql-dev openssl-dev libffi-dev jq pcre-dev python3-dev build-base linux-headers libxml2-dev libxslt-dev
+WORKDIR /usr/src/app
 
-RUN pip install --upgrade pip
-RUN pip install uwsgi supervisor
+# build deps
+RUN set -e && apt-get update && apt-get install --yes git gcc make postgresql-client libffi-dev python3-dev libxml2-dev libxslt-dev libpq-dev \
+                      nginx gettext arp-scan postgresql-client bash dbus
 
+RUN set -e && pip install --upgrade pip pipenv uwsgi supervisor
 # Install from lockfile
+# there is a bug in current pipenv version when using --system flag
 COPY Pipfile* ./
-RUN jq -r '.default | to_entries[] | .key + .value.version' \
-    Pipfile.lock > requirements.txt && pip install -r requirements.txt
-
-
-FROM python:3.7-alpine as runner
-RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+RUN set -e && pipenv lock -r > requirements.pip && pip install -r requirements.pip
+# build deps
+RUN apt-get remove --purge --yes gcc make postgresql-client libpq-dev libffi-dev python3-dev libxml2-dev libxslt-dev
```

Comment:
I'm pretty sure that the app will crash in runtime without `libxml2-dev` and `libxslt-dev`.

Reply:
Also, deleting this won't actually help as far as I can tell (but I can be wrong on this). The layer is already there previously.

Reply:
ok, I thought that it is required during build time only

Reply:
I put it back in [7535c8](https://github.com/fragaria/karmen/pull/213/commits/7535c8aee112e5b962f847f6ea5e9bdb86a40638)